snippets/field-trial.adoc

```diff
@@ -0,0 +1,10 @@
+// When including this file, ensure that {FeatureName} is set immediately before
+// the include. Otherwise it will result in an incorrect replacement.
+
+[IMPORTANT]
+====
+[subs="attributes+"]
+{FeatureName} is available as a Field Trial release. A Field Trial provides approved customers access to a release with general availability-level quality and limited support. For more information, contact link:https://www.redhat.com/en/contact[Red Hat Sales].
```

Comment:
Should you s/general availability-level quality/general-availability-level quality ??

Reply:
Just head back from Legal who changed this text so not an issue